### PR TITLE
[FIX] discuss: channel name overflow

### PR DESCRIPTION
**Pull request by maad-odoo**
Before this commit:
The channel name used to overflow in the channel suggestion section and also there was also no limit on the lenght of how long the name of the channel can be.

After this commit:
The name will not overflow in the channnel suggestion section and the user will also have a certain limit on the length of channel name:

Task id: 3366608

### CODE REVIEW

File: addons/web/static/src/scss/dropdown.scss [hunk 1]
```diff
@@ -9,6 +9,7 @@
     font-size: $font-size-base;
     padding: 5px 0px;
     box-shadow: 0px 6px 12px rgba(0, 0, 0, 0.176);
+    word-wrap: break-word;
```
Comment (alexkuhn) [inline addons/web/static/src/scss/dropdown.scss:12]: `ui-autocomplete` is used by other modules (e.g. website), and this likely might break current styling.
I prefer to play safe and apply this style in the custom class name that is used specifically in discuss code.

**Review by alexkuhn — COMMENTED** [review]
Note that In 16.3 we don't use `ui-autocomplete` anymore. Instead, this has become `NavigableList` component, which is a discuss-specific component.

**Comment by maad-odoo** [comment]
> Note that In 16.3 we don't use `ui-autocomplete` anymore. Instead, this has become `NavigableList` component, which is a discuss-specific component.

I looked into version 16.3 and noticed that the text-truncate class has already been added to the template. However, the text truncation is being restricted by the flex-shrink-0 property. It seems that applying a change in the `discuss.ChannelSelector.channel` template will solve the problem in version 16.3. So, may I apply change in its template for version 16.3?

**Comment by alexkuhn** [comment]
> I looked into version 16.3 and noticed that the text-truncate class has already been added to the template. However, the text truncation is being restricted by the flex-shrink-0 property. It seems that applying a change in the `discuss.ChannelSelector.channel` template will solve the problem in version 16.3. So, may I apply change in its template for version 16.3?

Yes, if this fixes the issue then this is good 🙂Just wanted to let you know this exact fix works until 16.2, and 16.3 will be quite different (although it's functionally the same resulting bug). Thanks!

**Comment by maad-odoo** [comment]
> > I looked into version 16.3 and noticed that the text-truncate class has already been added to the template. However, the text truncation is being restricted by the flex-shrink-0 property. It seems that applying a change in the `discuss.ChannelSelector.channel` template will solve the problem in version 16.3. So, may I apply change in its template for version 16.3?
> 
> Yes, if this fixes the issue then this is good 🙂Just wanted to let you know this exact fix works until 16.2, and 16.3 will be quite different (although it's functionally the same resulting bug). Thanks!

Done with the changes and also added the required PR. Please check 🙂.

**Review by alexkuhn — APPROVED** [review]
@robodoo r+
@fw-bot up-to saas-16.2

**Pull request closed without merging**